spec/jobs/ra/fetch_changes_batch_job_spec.rb

```diff
@@ -180,13 +216,18 @@
 
       district_change = Ra::DistrictChange.first
       expect(district_change).to have_attributes(
+        change_id: nil,
         district_id: 4764,
         municipality_id: 92,
         database_operation: 'INSERT',
         version_id: 1,
         created_reason: 'CREATE',
         valid_to: Time.parse('2004-04-30T23:59:59+02:00'),
-        unique_numbering: false
+        unique_numbering: false,
+
+        change: nil,
+        district_object: Ra::District.find(4764),
+        municipality: Ra::Municipality.find(92),
```

Comment:
`_object` som daval len k modelom naviazanym cez `object_id`.

Dalsou moznostou by bolo tie `object_id` asociacie spravit cez nieco ako `objekt_id` a `objekt`. Tym padom by sme to mali priblizne konzistentne a bez kolizii s Rails/Ruby keywordami.